### ISSUE TITLE
Disconnecting with AutoEat enabled crashes game

### ISSUE THREAD

**Issue by tycrek** [bebeli555/ElytraHighwayBot#1]
No matter the bot status, as long as it's in a "flying state" (flying, attempting to takeoff, etc) after `++start` or the Start button in the GUI, disconnecting from a server with AutoEat enabled crashes game. I was able  to replicate this in the 2b2t queue by opening the GUI, turning on AutoEat and Baritone (unsure if Baritone is required to make it crash), pressing Start, then disconnecting. Crash report follows:

```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  LoadingPlugin (Xaeros_Minimap_20.11.0_Forge_1.12.jar)
  ForgelinPlugin (Forgelin-1.8.4.jar)
  MixinLoaderForge (salhack-2.05-release.jar)
  ForgeHaxCoreMod (forgehax-1.12.2-2.9.0.jar)
  LoadingPlugin (ChunkAnimator-MC1.12-1.2.jar)
Contact their authors BEFORE contacting forge

// Hey, that tickles! Hehehe!

Time: 7/5/20 10:32 AM
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at com.bebeli555.ElytraBot.AutoEat.ShouldActivate(AutoEat.java:84)
	at com.bebeli555.ElytraBot.AutoEat.onUpdate(AutoEat.java:24)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_89_AutoEat_onUpdate_ClientTickEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.fml.common.FMLCommonHandler.onPostClientTick(FMLCommonHandler.java:349)
	at net.minecraft.client.Minecraft.func_71407_l(rj:1911)
	at net.minecraft.client.Minecraft.func_71411_J(rj:1098)
	at net.minecraft.client.Minecraft.func_99999_d(rj:398)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by bebeli555** [comment]
Oh yeah i forgot to turn the checks off when the player disconnects on AutoEat. Ill fix it in the next version thanks for the report
